extension/ts/content/entities/bookForm/util.ts

```diff
@@ -12,7 +12,9 @@ const getFormElements = (): FormAreaElement[] =>
 // This is relying on the fact that when the edit form is available, the html matches this selector,
 // and fails to match in all other cases. This IS brittle. If LibraryThing changes
 // the markup in any way this will just not work properly
-const formExists = (): boolean => !!document.querySelector("#book_editForm > .book_bit");
+const formExists = (): boolean =>
+	!!document.querySelector("#book_editForm > .book_bit") ||
+	!!document.querySelector("#book_editForm > #book_bookInformationSection > .book_bit");
```

Comment:
considering if we should make this a descendant selector instead of a strict child selector a la

`"#book_editForm .book_bit"`

Reply:
oh yeah that's way better